src/axom/slic/docs/sphinx/sections/architecture.rst

```diff
@@ -88,32 +88,37 @@ This indicates that all messages with a level of severity of ``WARNING`` and
 higher will be captured, namely ``WARNING`` and ``ERROR`` messages. Thereby,
 enable the application to filter out messages with lower severity.
 
-.. _logStream:
+.. warning::
+
+   No messages will be logged if you forget to call ``slic::setLoggingMsgLevel()``.
+   All messages will be ignored.
```

Comment:
Suggestion (minor):
```suggestion
   All messages will be ignored until the first call to ``slic::setLoggingMsgLevel()``
```

Reply:
Thanks for calling this out.

Is this really the design though? 
Seems `slic::initialize()` should default to some logging for its first logger, e.g. like logging warnings and errors.

Perhaps we should create an issue to discuss this?